src/include/platform/PlatformManager.h

```diff
@@ -354,9 +355,19 @@ inline void PlatformManager::UnlockChipStack()
     static_cast<ImplClass *>(this)->_UnlockChipStack();
 }
 
-inline void PlatformManager::PostEvent(const ChipDeviceEvent * event)
+inline CHIP_ERROR PlatformManager::PostEvent(const ChipDeviceEvent * event)
 {
-    static_cast<ImplClass *>(this)->_PostEvent(event);
+    return static_cast<ImplClass *>(this)->_PostEvent(event);
+}
+
+inline CHIP_ERROR PlatformManager::PostEventLoggingErrors(const ChipDeviceEvent * event)
```

Comment:
Why have two versions of this call, when the logging of the error could just be wrapped in a define? Do you really mean to have callers decide between the two? Think it would be more natural to just have logging of this be controlled through a global define.